.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature Request 
+about: Add a feature request
+---
+##  Before Logging a feature request
+[ ] Please search this repository's issues to see if it has already been reported. This helps reduce duplicate issues from being created.
+
+
+## Feature Request Summary 
+
+<!---Please provide a summary of the feature you request-->
+
+## Current Behavior

```

Comment:
not sure if this section is required for feature requests

Reply:
I put it as optional, what do you think?
<!---Optional: Please provide details about the desired behavior-->